install.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+# curl https://github.com/srevinsaju/zap/raw/main/install.sh | sh
+
```

Comment:
Add `set -euxo pipefail` to stop the script immediately when any single command fails.

Reply:
Fixed :)

removed all error checks. added pipefall.